src/Pact/Analyze/Util.hs

```diff
@@ -26,6 +26,9 @@ for2
   => s (t a) -> (a -> f b) -> f (s (t b))
 for2 = flip (traverse . traverse)
 
+foldFor :: (Foldable t, Monoid m) => t a -> (a -> m) -> m
+foldFor = flip foldMap
```

Comment:
hm. when I think "for" I typically think "imperative; effectful". where `for` is the flipped version of effectful `traverse`, rather than flipped (pure) map

Reply:
Great point. I just pushed a change to rename `foldFor` to `<&&>` and only define it locally. I'm not sure if this is better or not, but this is how I think of it:
* `<&&>` is analogous to `<&>` with an extra fold. I don't know of any common lens-like way to change a combinator to indicate a fold.
* This version reads well
* It's locally contained and easy to understand from the definition.

I'm happy to change the name or just use `flip foldMap` everywhere. Or maybe you can think of a better idea. But I think this seems good enough?